app/features/settings/metrics/SettingsMetricsPanel.js

```diff
@@ -111,6 +112,26 @@ class SettingsMetricsPanel extends Component {
         }
     }
 
+    _renderLastRow() {
+        let { height, width } = Dimensions.get('window');
+
+        if (width > 350) {
```

Comment:
- [ ] move this calculation to some utility class, make sure you do it for SetAnalysis as well